webpack.config.js

```diff
@@ -0,0 +1,25 @@
+const path = require('path')
```

Comment:
We have to add the webpack to `setup.py`. But I can do that later in a separate issue.

Reply:
@ziima:
Could you do it, please? I am not absolutely sure what you mean.
Also, could you resolve unresolved discussions or add reply on them, please?
Thanks